File: khalti-android/src/main/java/khalti/checkOut/api/ErrorAction.java
package khalti.checkOut.api;

public enum ErrorAction {
    WALLET_INITIATE("wallet_initiate"),
    WALLET_CONFIRM("wallet_confirm"),
    FETCH_BANK_LIST("fetch_bank_list"),
    FETCH_CARD_BANK_LIST("fetch_card_bank_list");

    private final String action;

    ErrorAction(String action) {
        this.action = action;
    }

    public String getAction() {
        return action;
    }
}

File: khalti-android/src/main/java/com/khalti/checkout/api/ErrorAction.kt
package com.khalti.checkout.api

enum class ErrorAction(val action: String) {
    WALLET_INITIATE("wallet_initiate"),
    WALLET_CONFIRM("wallet_confirm"),
    FETCH_BANK_LIST("fetch_bank_list"),
    FETCH_CARD_BANK_LIST("fetch_card_bank_list")
}